posthog/migrations/0257_add_insight_to_tiles_relationship_directly.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.14 on 2022-08-20 16:14
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0256_add_async_deletion_model"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="dashboardtile",
+            name="dashboard",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, related_name="dashboard_tiles", to="posthog.dashboard"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="dashboardtile",
+            name="insight",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, related_name="dashboard_tiles", to="posthog.insight"
+            ),
+        ),
```

Comment:
Flyby, if we're dealing with migrations that don't actually alter the database, just do some django-cleanup, it'd be bread to explicitly add a `# actually a no-op` or such around it.